Mathlib/CategoryTheory/Bicategory/Modification/Oplax.lean

```diff
@@ -83,37 +87,22 @@ theorem whiskerRight_naturality (f : a ⟶ b) (g : G.obj b ⟶ a') :
 
 end
 
-/-- Vertical composition of modifications. -/
-@[simps]
-def vcomp (Γ : Modification η θ) (Δ : Modification θ ι) : Modification η ι where
-  app a := Γ.app a ≫ Δ.app a
-
 end Modification
 
 /-- Category structure on the oplax natural transformations between OplaxFunctors. -/
 @[simps]
-scoped instance category (F G : OplaxFunctor B C) : Category (F ⟶ G) where
+scoped instance homCategory (F G : OplaxFunctor B C) : Category (F ⟶ G) where
   Hom := Modification
-  id := Modification.id
-  comp := Modification.vcomp
+  id η := { app := fun a ↦ 𝟙 (η.app a) }
+  comp Γ Δ := { app := fun a => Γ.app a ≫ Δ.app a }
```

Comment:
It seems that the original ones with `@[simps!]` instead of `@[simps]` work. The `simps!` produces `id_app'` and `comp_app'` automatically.

Reply:
Yes the original ones also work, I just moved them inside the category instance since they should be accessed using the category theoretic notation anyways. Do you think this is a mistake?

Reply:
I don't think it's a mistake, but I think it's a matter of preference. My own rule of thumb is that if a notion has a certain mathematical content, then it's worth declaring it with `def` (or at least `abbrev`). The `id` and `vcomp` are worth declaring in my opinion.

Reply:
Okay I agree now, fixed!